README.md

```diff
@@ -1,2 +1,180 @@
-# Rocket.Chat.App-RapidPro
-Integration between RocketChat Omnichannel (LiveChat) and RapidPro platform
+# [APP_NAME]
+
+## About
+
+[APP_NAME] is a [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) app to enable the integration between [rapidpro](https://github.com/rapidpro/rapidpro) ticketing service and the Rocket.Chat Omnichannel(Livechat) feature.
```

Comment:
Maybe more evidence here about the platform that this app will integrate. Rapidpro, Textit or Push.

What do you think to use a bulleted list or bold text in platforms' names?